src/PrometheusPushGateway/PushGateway.php

```diff
@@ -25,12 +25,12 @@ class PushGateway
 
     /**
      * PushGateway constructor.
-     * @param string $address host:port of the push gateway
+     * @param string $address http|https://host:port of the push gateway
```

Comment:
maybe easier to read if we do `(http|https)://host:port`?

Reply:
very sensible suggestion